### PR TITLE
Cardahub.json

### PR DESCRIPTION
**Pull request by clawoo**

### CODE REVIEW

File: dApps/Cardahub.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
             ]
         },
         {
-            "name": "???",
+            "name": "Cardahub.io's collection",
```
Comment (matiwinnetou) [inline dApps/Cardahub.json:27]: Does this collection have any particular name?

Comment (clawoo) [reply dApps/Cardahub.json:27]: This is the name mentioned on adapools in the collection section

Comment (matiwinnetou) [reply dApps/Cardahub.json:27]: yes.. because we will have:
```
Cardahub - Cardahub.io's collection
```

we need to name this in some other way

Comment (clawoo) [reply dApps/Cardahub.json:27]: Maybe just change it to collection or CNFT collection

**Pull request closed without merging**